backend/de.metas.business/src/main/java/de/metas/product/impl/ProductPlanningSchemaDAO.java

```diff
@@ -154,7 +223,7 @@ public static void save(@NonNull final ProductPlanningSchema schema)
 	/**
 	 * @return All the active products with the given product planning schema selector
 	 */
-	public static Set<ProductId> retrieveProductIdsForSchemaSelector(
+	public static Set<ImmutablePair<ProductId, OrgId>> retrieveProductIdsForSchemaSelector(
```

Comment:
consider moving this method to ProductDAO

Reply:
done